packages/component-library/src/CivicStoryCard/CivicStoryLink.js

```diff
@@ -29,11 +29,13 @@ const storyLinkClass = css`
   }
 `;
 
-const StoryLink = ({ children, icon, route, action }) => (
+const StoryLink = ({ children, icon, route, action, link }) => (
   <div className={storyLinkClass}>
     {route
       ? <Link to={route}><i className={icon} /><span>{children}</span></Link>
-      : <a tabIndex="0" onClick={action}><i className={icon} /><span>{children}</span></a>
+      : link
+        ? <a href={link}><i className={icon} /><span>{children}</span></a>
+        : <a tabIndex="0" onClick={action}><i className={icon} /><span>{children}</span></a>
```

Comment:
It's inconsequential, but I like to format nested ternaries this way. It prevents the awkward indentation issue and reads like question/answer phrases per line.

```jsx
{
  route ? <Link to={route}><i className={icon} /><span>{children}</span></Link> :
  link ? <a href={link}><i className={icon} /><span>{children}</span></a> :
  <a tabIndex="0" onClick={action}><i className={icon} /><span>{children}</span></a>
}